docs/introduction.md

```diff
@@ -0,0 +1,232 @@
+# Introduction
+
+This document is aimed at introducing developers to the Flow Data Provisioning Service project.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+3. [What is Flow](#flow)
+    1. [Glossary](#glossary)
+4. [Developer Guide](#developer-guide)
+    1. [Installation](#installation)
+    2. [Setting up a test environment](#setting-up-a-test-environment)
+5. [More resources](#more-resources)
+
+## Getting Started
+
+The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow [execution state](#execution-nodes).
+
+The reason for this need is that in the execution state, blocks are pruned after a hundred other blocks have been finalized, which makes it impossible to visualize the history of the chain.
+By providing an API which exposes the Flow chain history and implementing the widely used [Rosetta API specification](https://www.rosetta-api.org/), it becomes possible to integrate the Flow blockchain into many 3rd party applications and tools.
+
+When the Flow blockchain is upgraded with a new version that introduces changes in [pathfinding](#merkle-patricia-tries), the old chain gets [_sporked_](#sporks) and a new one gets created based on the state of the old one.
+The old version of the chain is maintained as a static chain which will never change, with just one [execution node](#execution-nodes) and one access node to provide read access into the chain's state.
```

Comment:
I would change it slightly with something like "the final version of the previous execution state remains available through a legacy access node that connects to a legacy execution node". You could also clarify that this is only for the last 300 chunks.